Machine Learning/sklearn examples/Decision Trees and Random Forests/DecisionTrees_and_RandomForests.ipynb

```diff
@@ -0,0 +1,529 @@
+{
```

Comment:
<p>Instead of "The end", could you have a small conclusion could even be just 2 lines summarizing what we did in the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
I have made the requested change. Please check and let me know if anything else is needed. I was a little caught up in some work, so it took quite a lot of time...